10_08_2017/perturbed_big_disp.py

```diff
@@ -0,0 +1,453 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 22 13:17:02 2016
+
+@author: app13jfm
+"""
+
+from __future__ import division
+#import matplotlib
+import numpy as np
+#import matplotlib.cm as cm
+#import matplotlib.mlab as mlab
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+#import math as mt
+import mpmath as mp
+import scipy.optimize as sp
+#from mpl_toolkits.mplot3d import Axes3D
+#from mpl_toolkits.axes_grid1 import host_subplot
+#from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+#from mpl_toolkits.axes_grid1.inset_locator import mark_inset
+import matplotlib.gridspec as gridspec
+import matplotlib.patches as mpatches
+import matplotlib.lines as mlines
+from mpl_toolkits.mplot3d.axes3d import Axes3D
+#from matplotlib import rc
+#import pdb
+#import pickle
+gs = gridspec.GridSpec(2, 1,height_ratios=[3,1])
+mp.dps=10
+mp.pretty= True
+
+import mpmath as mp
+
+
+d = 2
+theta = 0.1
+
+g=5/6
+
+G=10
+
+#define gammas
+def gamma_0_a0(W):
+    return (-G * W**2 + np.sqrt( G**2 * W**4 - 4*(W**2 - b) * (W**2 - 1) * ((1+b) * W**2 - b) +0j)) / (2 * ((1+b) * W**2 - b) + G * W**2)
+def gamma_1_a0(W):
+    g0 = gamma_0_a0(W)
+    return 1j * (-2 * b * g0**3 + G * g0**2 + 2 * b * g0 + G) / (2 * ((1+b) * W**2 - b) * g0 + G * W**2)
```

Comment:
(Convention)[https://www.python.org/dev/peps/pep-0008/] is two blank lines between function definitions. With that plus the number of functions you're defining they should probably be defined in their own script and imported in.